packages/workspace/src/generators/preset/preset.ts

```diff
@@ -140,12 +140,6 @@ async function createPreset(tree: Tree, options: Schema) {
   }
 }
 
-function addPolyfills(host: Tree, polyfillsPath: string, polyfills: string[]) {
```

Comment:
This seems unrelated. Why was this removed?

Reply:
Unused